_data/public-speaking.yml

```diff
@@ -14,3 +14,23 @@
   duration: "40m"
   image: "assets/images-events/Public-Speaking-Modern JS.png"
   linkedIn: "https://www.linkedin.com/posts/itenium_itenium-modernjs-publicspeaking-activity-7178345170629738496-LcR0?utm_source=share&utm_medium=member_desktop"
+
+- name: "React Brussels & MitTechCon"
+  by: "Wouter"
+  event: "React Brussels & MitTechCon"
+  date: 2024-03-21 
+  summary: >
+    Wouter Van Schandevijl of Itenium gave a detailed talk on 
+    March 21st in Detroit focused on React component profiling. 
+    He broke down how React's rendering process works, demonstrated 
+    the use of React DevTools Profiler for identifying performance 
+    bottlenecks, and shared practical strategies for optimizing 
+    component behavior. The session was geared towards developers, 
+    offering concrete techniques to make React apps faster and more 
+    efficient.
+  content: "Theoretical"
+  level: Introduction
+  duration: "1h"
+  image: "assets/images-events/React-Component-profiling-1.jpeg"
+  linkedIn: "https://www.linkedin.com/posts/itenium_itenium-react-performanceoptimization-activity-7176171038290931713-r3mI?utm_source=share&utm_medium=member_desktop"
```

Comment:
I gave a comment on the LinkedIn urls

If I say something and you think you could EVER EVER EVER forget about this:
Add it to the instructions, add it to the PR checklist.

I do not want to keep repeating myself, I don't want to keep saying the same thing every PR....